templates/jobsystem.mustache

```diff
@@ -59,15 +59,138 @@
+        '</div>' +
+        '</div>'],
+        ['TCA', 29.95602, -90.10512,
+        '<div id="content">' +
+        '<div id="siteNotice">' +
+        '</div>' +
+        '<h2>Total Community Action</h2>' +
+        '<div id="bodyContent">' +
+        '<p>Mon-Fri, 8AM - 5PM</p>' +
+        '<p>1424 S. Jefferson Davis Parkway, New Orleans, LA 70125</p>' +
+        '<p><a href="https://goo.gl/maps/eq5h2oCM2f12" target="_blank" title="TCA address in Google Maps">Open in Google Maps</a></p>' +
+        '<p><a href="tel:+5048720334">(504) 872-0334</a></p>' +
+        '<p><a href="http://www.tca-nola.org/" target="_blank" title="TCA Website">Visit the website</a></p>' +
+        '</div>' +
+        '</div>'],
+        ['Urban League', 29.97723, -90.03875,
+        '<div id="content">' +
+        '<div id="siteNotice">' +
+        '</div>' +
+        '<h2>Urban League</h2>' +
+        '<div id="bodyContent">' +
+        '<p>Mon-Fri, 8AM - 5PM</p>' +
+        '<p>3232 North Galvez Street, New Orleans, LA 70117</p>' +
+        '<p><a href="https://goo.gl/maps/aWS8QZrKGYn" target="_blank" title="Urban League address in Google Maps">Open in Google Maps</a></p>' +
+        '<p><a href="tel:+5043244444">(504) 324-4444</a></p>' +
+        '<p><a href="http://www.urbanleagueneworleans.org/ul/" target="_blank" title="Urban League Website">Visit the website</a></p>' +
+        '</div>' +
+        '</div>']
+      ],
+      mapCenter = new google.maps.LatLng(29.97036, -90.07493),
+      mapCanvas = document.getElementById('map_canvas'),
+      mapOptions = {
+        zoom: 13,
+        center: new google.maps.LatLng(0, 0),
+        mapTypeId: google.maps.MapTypeId.ROADMAP
+      },
+      map = new google.maps.Map(mapCanvas, mapOptions),
+      infowindow = new google.maps.InfoWindow(),
+      bounds = new google.maps.LatLngBounds(),
+      marker, i;
+      for (i = 0; i < locations.length; i++) {
+        var myLatLng = new google.maps.LatLng(locations[i][1], locations[i][2]);
+        marker = new google.maps.Marker({
+          position: new google.maps.LatLng(locations[i][1], locations[i][2]),
+          map: map
+        });
+        google.maps.event.addListener(marker, 'click', (function(marker, i) {
+          return function() {
+            infowindow.setContent(locations[i][3]);
+            infowindow.open(map, marker);
+          }
+        })(marker, i));
+        bounds.extend(myLatLng);
+      };
+      map.fitBounds(bounds);
+      return {
+        init: function() {
+          map.set('styles', [{
+            featureType: 'landscape',
+            elementType: 'geometry',
+            stylers: [{
+              hue: '#ffff00'
+            }, {
+              saturation: 30
+            }, {
+              lightness: 10
+            }]
+          }]);
+
+        }
+      };
+    }());
+
+    bittersMap.init();
+    </script>
+  </div>
+</section>
```

Comment:
If anyone wants to shave a yak, it would be good to move the map code into a partial since it's used in more than one place now. But not necessary.